drivers/scheduler/k8s/k8s.go

```diff
@@ -2929,6 +2936,55 @@ func (k *K8s) getExpectedAutopilotSizeForPvc(pvc *v1.PersistentVolumeClaim, apRu
 	return expectedPVCSize, nil
 }
 
+// ValidateEvents verifies proper alerts and events on resize completion
+func (k *K8s) ValidateEvents(pvc *v1.PersistentVolumeClaim, apRules *apapi.AutopilotRuleList) error {
+
+	// wait until autopilot posts all events
+	time.Sleep(time.Minute)
+
+	successEvens := []string{
+		"Initializing => Normal",
+		"Normal => Triggered",
+		"Triggered => ActiveActionsPending",
+		"ActiveActionsPending => ActiveActionsInProgress",
+		"ActiveActionsInProgress => ActiveActionsTaken",
+	}
+
+	for _, apRule := range apRules.Items {
+		if isAutopilotMatchPvcLabels(apRule, pvc) {
+
+			fields := fmt.Sprintf("involvedObject.kind=%s,involvedObject.name=%s", "AutopilotRule", apRule.Name)
+			events, err := k8sops.Instance().ListEvents("", metav1.ListOptions{FieldSelector: fields})
+			if err != nil {
+				return err
+			}
+
+			eventMap := make(map[string]bool)
+			for _, event := range events.Items {
+				if time.Since(event.CreationTimestamp.Time).Hours() > float64(1) {
+					logrus.Infof("Can't capture all events for %s:%s, since resize has started more that 1h ago", apRule.Name, pvc.Name)
+					return nil
+				}
+
+				// cut last 3 entries from a message like `rule: pvc:pvc-25c transition from ActiveActionsInProgress => ActiveActionsTaken`
```

Comment:
how about we  do a more loose strings.Contains check instead of cutting the last 3 entries?

Reply:
ideally we need to change events, because validation, based on messages is not reliable. 
At this point, to keep validation simple, I'm using this approach (I'm actually building the message again). Otherwise the logic will be too complicated.

I'll create a AUT to modify events to be more suitable for testing